samples/sensor/amg88xx/README.rst

```diff
@@ -0,0 +1,69 @@
+.. _amg88xx:
+
+AMG88XX Infrared Array Sensor
+#############################
+
+Overview
+********
+
+Sample application that periodically reads 8x8 temperature array from
```

Comment:
How about:

     This sample application periodically reads the 8x8 temperature array from
     the AMG88XX sensor, and displays its values to the serial console.